index.bs

```diff
@@ -316,56 +317,90 @@ Interface {#interface}
 =================
 
 <pre class="idl">
-dictionary NavigatorUABrandVersionDict {
-  required DOMString brand;
+dictionary NavigatorUABrandVersion {
+  DOMString brand;
   DOMString version;
```

Comment:
Is this the full version, or the major version? If the former, the latter should be in the high-entropy group. If the latter, should we expose the former distinctly so developers can make a choice about what data they access above and beyond what we give them by default?

Reply:
This is the major version. I intend to add a full version API (to the high entropy group) in a followup